third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -1859,30 +1859,37 @@ function getNodeBoundingClientRects(nodeRrpId) {
  * @see https://static.replay.io/protocol/tot/DOM/#type-BoxModel
  */
 function DOM_getBoxModel({ node: nodeRrpId }) {
-  const nodeId = getBlinkNodeIdByRrpId(nodeRrpId);
-  /**
-   * @see https://chromedevtools.github.io/devtools-protocol/tot/DOM/#type-BoxModel
-   */
-  const cdpModel = fromJsGetBoxModel(nodeId);
+  const nodeObj = getPlainObjectByRrpId(nodeRrpId);
 
   const model = {
     node: nodeRrpId
   };
 
-  if (cdpModel) {
-    const {
-      content, padding, border, margin,
-      // width, height, shapeOutside
-    } = cdpModel;
-    Object.assign(
-      model,
-      {
-        content,
-        padding,
-        border,
-        margin
-      }
-    );
+  if (!isBlinkInstanceOf(nodeObj, Element)) {
+    // Handle invalid input (this is how we do this in Gecko)
+    model.content = [];
```

Comment:
@Domiii This doesn't look like it fulfills the protocol api. May need to add the other fields with empty arrays either here or above on L1865.

```
  # Get boxes for a node.
  command getBoxModel
    parameters
      # Node to get boxes for.
      Pause.ObjectId node
    returns
      # Box model data for the node.
      BoxModel model

  # Description of the box model for a node.
  type BoxModel extends object
    properties
      # Node this is the box model for.
      Pause.ObjectId node
      # Quads holding the node's content.
      Quads content
      # Quads including the content and any padding between the border.
      Quads padding
      # Quads including the border.
      Quads border
      # Quads including the border and any margin between other nodes.
      Quads margin
```

Reply:
Thanks for checking the protocol. Of course I was too lazy to do so myself 👀 
I'll add it in a follow-up!